crates/subspace-runtime/build.rs

```diff
@@ -18,14 +18,22 @@ use std::env;
 use substrate_wasm_builder::WasmBuilder;
 
 fn main() {
-    let cargo_manifest_dir = env::var("CARGO_MANIFEST_DIR").expect("Set by cargo; qed");
+    let relative_target_dir = env!("CARGO_MANIFEST_DIR").to_owned() + "/../../target";
```

Comment:
Why changing from `std::var()` to `env!()`? It is typically not a good idea to use macros here due to unexpected behavior in compilation vs when build script is running.

Reply:
Why is it unexpected though? I just checked and for build scripts in also force to recompile a build script if environment variable was changed, so this behaviour is more expected then using `std::env::var` without [`println!("cargo:rerun-if-env-changed=...")](https://doc.rust-lang.org/cargo/reference/build-scripts.html#cargorerun-if-env-changedname)

Reply:
I have not found anything that suggest `cargo:rerun-if-env-changed` being added with macro instead of `std::env::var()` call, I was primarily recalling the beginning of https://doc.rust-lang.org/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-build-scripts

Did you verify it still works without `CARGO_TARGET_DIR`?

Reply:
Yep, it works if it is set or unset